mbf_abstract_nav/src/abstract_controller_execution.cpp

```diff
@@ -348,13 +346,21 @@ void AbstractControllerExecution::run()
       }
 
       // compute robot pose and store it in robot_pose_
-      computeRobotPose();
+      if (!computeRobotPose())
+      {
+        publishZeroVelocity();
```

Comment:
we should put all these lines in a setFailureState method...  I'll do in another PR